src/content/smart-bulletins/038.md

```diff
@@ -0,0 +1,96 @@
+(emphasis added) agrees, without statutory authorization or other exception, to an
+open-ended, unrestricted indemnification clause.
+
+On April 4, 2013, the Office of Management and Budget (OMB) issued guidance
+outlining a series of management actions to ensure agencies act in compliance
+with the ADA and in accordance with OLC's opinion. This guidance clarify that the
+inclusion of an open-ended indemnification clause in an End User License
+Agreement (EULA), online Terms of Service (TOS), or other agreement, is not
+binding on the Government unless expressly authorized by statute and specifically
+authorized under applicable agency regulations and procedures, and shall be
+deemed to be stricken from the EULA, TOS, or similar legal instrument or
+agreement.
+
+The GSA SmartPay 3 Master Contract includes Federal Acquisition Regulation
+(FAR) Clause 52.212-4, entitled “Contract Terms and Conditions - Commercial
+Items.” In addition, GSA SmartPay Master Contract Modification number 19 added
+GSA Acquisition Manual (GSAM) clause 552.232-39, entitled “Unenforceability of
+Unauthorized Obligations (FEB 2018) (DEVIATION FAR 52.232-39)” and GSAM
+Clause 552.232-78, “Commercial Supplier Agreements–Unenforceable.”
+
+These GSA SmartPay 3 Master Contract clauses address situations when there is
+an unrestricted, open-ended indemnification provision in EULA, TOS, or similar
+legal instruments or agreements. The clauses clarify that if a EULA, TOS, or
+similar legal instrument or agreement includes a requirement for the Government
+to indemnify the contractor or any person or entity for damages, costs, or fees, or
+any other loss or liability that would create an Anti-Deficiency Act violation, such
+requirements are unenforceable against the Government, and are deemed to be
+stricken from the agreement to prevent violations of the Anti-Deficiency Act.
+
+It is important to note that the GSA SmartPay 3 Master Contract clauses
+referenced above are not intended to restrict the contractor bank from requiring
+government end users to sign account holder applications, agree to the
+submission of applications and/or account holder agreements. Government end
+users are still required to agree to the terms of these documents so long as they
+comply with the GSA SmartPay 3 master contracts and do not create
+Anti-Deficiency violations.
+
+The following links provide additional information on applicable clauses and
+guidance pertaining to the Unenforceability of Unauthorized Obligations:
+
+GSAM Clause 552.232-39, “Unenforceability of Unauthorized
+Obligations”:[https://www.acquisition.gov/gsam/552.232-39](https://www.acquisition.gov/gsam/552.232-39)
+
+GSAM Clause 552.232-78, “Commercial Supplier
+Agreements–Unenforceable”:[https://www.acquisition.gov/gsam/552.232-78](https://www.acquisition.gov/gsam/552.232-39)
+
+## Action
+
+This Smart Buletin is provided to AOPCs in responding to questions from
```

Comment:
This Smart Buletin is provided to AOPCs in responding to questions from cardholders regarding the applicability of contractor bank commercial EULAs, TOS, and other “click to agree ” agreements to Federal government accounts and cardholders.
It is important for agencies to review their current card policies and guidance to ensure that they are updated to reflect the information contained in this Smart Bulletin.